src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,42 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+/**
+ * Represents the tag added/removed from a Person
+ */
+public class Tagging {
+
+    private boolean isAdd; // Assume that there is only add or remove command for tagging
```

Comment:
A boolean variable works here, but an `enum` may be more appropriate because the two values we need are `added` and `deleted`, not `true` and `false`.